content/en/docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -88,11 +88,14 @@ directly to the API server, like this:
 Using `grep/cut` approach:
 
 ``` shell
```

Comment:
For this sample to render correctly, this white space needs to be removed:
<code>```shell</code>

Reply:
@aqnaqn 👋 Thanks for this PR! This needs a one-character change before it can merge.